R/pkg/R/mllib.R

```diff
@@ -292,6 +299,44 @@ setMethod("summary", signature(object = "NaiveBayesModel"),
             return(list(apriori = apriori, tables = tables))
           })
 
+#' Isotonic Regression Model
+#' Fits an Isotonic Regression model against a Spark DataFrame, similarly to R's isoreg().
+#' Users can print, make predictions on the produced model and save the model to the input path.
+#'
+#' @param data SparkDataFrame for training
+#' @param formula A symbolic description of the model to be fitted. Currently only a few formula
+#'                operators are supported, including '~', '.', ':', '+', and '-'.
+#' @param isotonic Whether the output sequence should be isotonic/increasing (true) or
+#'                 antitonic/decreasing (false)
+#' @param featureIndex The index of the feature if \code{featuresCol} is a vector column (default: `0`),
+#'                     no effect otherwise
+#' @return \code{spark.isotonicRegression} returns a fitted Isotonic Regression model
+#' @rdname spark.isotonicRegression
+#' @name spark.isotonicRegression
+#' @export
+#' @note spark.isotonicRegression since 2.1.0
+#' @seealso \link{transform}, \link{read.ml}, \link{write.ml}
```

Comment:
I don't think we put @seealso to transform/read/write ml here?

Also you should add @alias like this
`#' @aliases spark.naiveBayes,SparkDataFrame,formula-method`